db/migrate/20170709195655_migrate_countries.rb

```diff
@@ -0,0 +1,63 @@
+class MigrateCountries < ActiveRecord::Migration[5.1]
+  CHANGES = {
+    "Aruba" => "Netherlands",
+    "Korea" => "Republic of Korea",
+    "USA" => "United States",
+    "Puerto Rico" => "United States"
+  }
+
+  def up
+    Country.transaction do
+      import_all_missing_countries
+
+      CHANGES.each do |old_name, new_name|
+        puts "Renaming #{old_name} to #{new_name}"
+
+        old_country = Country.find_by(name: old_name)
+        next if old_country.nil?
```

Comment:
Probably never.. It's a bit defensive I guess. My thought was someone else could use this app with a Country table that's already identical to the WCA database. For that person, this migration would otherwise fail. But realistically I think I'm the only person running this app :-)